File: Core/Models/Dtos/GameInstanceDto.cs
using System;
using System.Collections.Generic;

namespace Core.Models.Dtos
{
    public class GameInstanceDto
    {
        public Guid Id { get; set; }
        public int GameId { get; set; }
        public DateTimeOffset DateCreated { get; set; }

        public GameInstanceStateDto State { get; set; }
        public ICollection<GameInstanceUserDto> Users { get; set; } = new List<GameInstanceUserDto>();
    }
}

File: UI.Web/ClientApp/src/app/models/GameInstanceDto.ts
import { GameInstanceStateDto } from "./GameInstanceStateDto";
import { GameInstanceUserDto } from "./GameInstanceUserDto";

export class GameInstanceDto {
    constructor(){
    }
    id: number;
    gameId: number;
    dateCreated: Date;
    gameDisplayName: string;

    state: GameInstanceStateDto;
    users: GameInstanceUserDto[];
}